python/paddle/fluid/dygraph/jit.py

```diff
@@ -525,6 +528,58 @@ def _build_load_path_and_config(path, config):
     return model_path, config
 
 
+_save_pre_hooks_lock = threading.Lock()
+_save_pre_hooks = []
+
+
+class HookRemoveHelper(object):
+    """ A HookRemoveHelper that can be used to remove hook. """
+
+    def __init__(self, hook):
+        self._hook = hook
+
+    def remove(self):
+        remove_save_pre_hook(self._hook)
+
+
+def register_save_pre_hook(hook):
+    global _save_pre_hooks_lock
+    global _save_pre_hooks
+    _save_pre_hooks_lock.acquire()
+    if hook not in _save_pre_hooks:
+        _save_pre_hooks.append(hook)
+    _save_pre_hooks_lock.release()
+    return HookRemoveHelper(hook)
+
+
+def remove_save_pre_hook(hook):
```

Comment:
if `remove_save_pre_hook` is not used as public api, we recommoned nameing it with prefix `_` ,  `_remove_save_pre_hook`

Reply:
Done, renamed `remove_save_pre_hook` to `_remove_save_pre_hook `